api/src/search/controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get } from "routing-controllers";
+
+import { GetSearchResponse } from "./types";
+import { SearchService } from "./service";
+import { Service } from "typedi";
+
+@Service()
+@Controller("/Search")
+export class SearchController {
+  constructor(private readonly searchService: SearchService) {}
+
+  @Get("/")
+  public async search(): Promise<GetSearchResponse> {
+    const searchResults = await this.searchService.search("project", "test");
```

Comment:
We should get results for all models not just project:

```suggestion
    const searchResults = await this.searchService.search("test");
```